diff --git a/dist/shaka-player.compiled.d.ts b/dist/shaka-player.compiled.d.ts
index 53815afc3d93759b9bfb1ce0fdfce7e8438c654f..8e23f5a00e0ce5a767fad81d733f474691b263f0 100644
--- a/dist/shaka-player.compiled.d.ts
+++ b/dist/shaka-player.compiled.d.ts
@@ -5364,3 +5364,5 @@ declare namespace shaka.extern {
 declare namespace shaka.extern {
   type TransmuxerPlugin = ( ) => shaka.extern.Transmuxer ;
 }
+
+export default shaka;
diff --git a/dist/shaka-player.compiled.debug.d.ts b/dist/shaka-player.compiled.debug.d.ts
index b3f5b6f2c9472bb49ae046eb2c2451668daf87a0..601bf0a717a690fe87de5aea03a6fbc349a0bfb7 100644
--- a/dist/shaka-player.compiled.debug.d.ts
+++ b/dist/shaka-player.compiled.debug.d.ts
@@ -5364,3 +5364,5 @@ declare namespace shaka.extern {
 declare namespace shaka.extern {
   type TransmuxerPlugin = ( ) => shaka.extern.Transmuxer ;
 }
+
+export default shaka;
diff --git a/dist/shaka-player.dash.d.ts b/dist/shaka-player.dash.d.ts
index 66fcede1c43dafc0d8e55162bcdb84338eee3bb8..283cdc78e2af9498517814a5cc2bcc5b5ee1e09a 100644
--- a/dist/shaka-player.dash.d.ts
+++ b/dist/shaka-player.dash.d.ts
@@ -4340,3 +4340,5 @@ declare namespace shaka.extern {
 declare namespace shaka.extern {
   type TransmuxerPlugin = ( ) => shaka.extern.Transmuxer ;
 }
+
+export default shaka;
diff --git a/dist/shaka-player.dash.debug.d.ts b/dist/shaka-player.dash.debug.d.ts
index 133ebaadeff5266f7c8f90d776a9f38068ad0875..f7e6d4b0ced5f5b515d0d5d5e35e8474d6b31697 100644
--- a/dist/shaka-player.dash.debug.d.ts
+++ b/dist/shaka-player.dash.debug.d.ts
@@ -4340,3 +4340,5 @@ declare namespace shaka.extern {
 declare namespace shaka.extern {
   type TransmuxerPlugin = ( ) => shaka.extern.Transmuxer ;
 }
+
+export default shaka;
diff --git a/dist/shaka-player.hls.d.ts b/dist/shaka-player.hls.d.ts
index 544f72d480b67480fbfad844003f13c985ddd4f7..23cb0e02c3dba161c6ab4cdf24bc92410167c306 100644
--- a/dist/shaka-player.hls.d.ts
+++ b/dist/shaka-player.hls.d.ts
@@ -4437,3 +4437,5 @@ declare namespace shaka.extern {
 declare namespace shaka.extern {
   type TransmuxerPlugin = ( ) => shaka.extern.Transmuxer ;
 }
+
+export default shaka;
diff --git a/dist/shaka-player.hls.debug.d.ts b/dist/shaka-player.hls.debug.d.ts
index 78055db34c85058fd04a41258c59c26a190ea78c..2101f7e1bbc7614f5d547a78f65a13c74fbe9b40 100644
--- a/dist/shaka-player.hls.debug.d.ts
+++ b/dist/shaka-player.hls.debug.d.ts
@@ -4437,3 +4437,5 @@ declare namespace shaka.extern {
 declare namespace shaka.extern {
   type TransmuxerPlugin = ( ) => shaka.extern.Transmuxer ;
 }
+
+export default shaka;
diff --git a/dist/shaka-player.ui.d.ts b/dist/shaka-player.ui.d.ts
index 4f23ea25d04e7573ce77dd6bfdae2820514ff37d..bea990967c5e424efe9748dc87ab7b834daebaf8 100644
--- a/dist/shaka-player.ui.d.ts
+++ b/dist/shaka-player.ui.d.ts
@@ -6110,3 +6110,5 @@ declare namespace shaka.extern {
 declare namespace shaka.ui.Watermark {
   type Options = { alpha : number , color : string , displayDuration : number , interval : number , position : string , size : number , skip : number , text : string , transitionDuration : number , type : string } ;
 }
+
+export default shaka;
diff --git a/dist/shaka-player.ui.debug.d.ts b/dist/shaka-player.ui.debug.d.ts
index 8f714263b795c1cb17f8b2daa54ecaccbfb31baa..d2f822b6d3545ec5095595fba315f545116bc09d 100644
--- a/dist/shaka-player.ui.debug.d.ts
+++ b/dist/shaka-player.ui.debug.d.ts
@@ -6110,3 +6110,5 @@ declare namespace shaka.extern {
 declare namespace shaka.ui.Watermark {
   type Options = { alpha : number , color : string , displayDuration : number , interval : number , position : string , size : number , skip : number , text : string , transitionDuration : number , type : string } ;
 }
+
+export default shaka;
